controller/src/beerocks/master/db/db.h

```diff
@@ -726,6 +739,22 @@ class db {
 
     std::shared_ptr<vaps_list_t> m_vap_list;
 
+    /*
+    * This map holds link metric "data struct" per reporting Agent sMacAddr .
+    * "data struct" holds map of the actual link_metrics_data vector (tx/rx) per reported Agent sMacAddr.
+    * Map is Used in TYPE_GW/TYPE_IRE nodes.
+    * Map created empty in all other nodes.
+    */
+    std::unordered_map<sMacAddr, std::unordered_map<sMacAddr, son::node::link_metrics_data>>
+        m_link_metric_data;
+
+    /*
+    * This map holds ap metric data per reporting Agent sMacAddr .
+    * Map is Used in TYPE_GW/TYPE_IRE nodes.
+    * Map created empty in all other nodes.
+    */
```

Comment:
Add a TODO that this should move to the BSS nodes (which currently don't exist) instead of being a separate map.

Reply:
Done in #498 .